docs/Best Practice Documents/2.2. Launch by Voice Recognition/index.md

```diff
@@ -0,0 +1,72 @@
+# 2.2. Launch by Voice Recognition
+
+## 1. Overview
+This chapter describes the launch of the SDL App by Voice Recognition (pushing Push To Talk (PTT) button).
+
+## 2. Background/Purpose/Reason for Standardization
+Currently, the launch of the SDL App is an SDL standard behavior. However, since triggers that launch the SDL App are not explicitly defined in the SDL standard specification, it is necessary for the OEMs to define it by themselves. Hence, the purpose of this document is to standardize such cases/issues using the TOYOTA specification, in order to be able to contribute to the SDL Ecosystem.
+
+## 3. Function Details
+### 3.1. Function Overview
+The user operations that trigger the launch of SDL App are listed below:
+
+&ensp;(1) By pressing the SDL App Icon
+&ensp;(2) By launching Voice Recognition
+&ensp;(3) By choosing the Audio source on the Audio screen
+&ensp;(4) By pressing the Hard SW
+&ensp;(5) By pressing the ModeSW on the Steering Wheel
+
+The following (1), (3), (4), (5) mentioned above, already each has of their own documents. Thus, this chapter provides information about (2).
+
+### 3.2. Launch the SDL App by Voice recognition
+By short press of the PTT button, voice recognition is activated, and the user can launch and display the SDL App on HU by recognizing the SDL App name. Even after launching the SDL App, you can still continue to use Voice Recognition. The Voice Recognition is canceled, when it detects a voice key word or a silence during the voice recognition state. In addition, if the HU receives a launching request of the SDL App by Voice Recognition, then the HU will decide that the SDL App can not be launched, (in this case, when the mobile is not connected or the SDL app is not registed, etc.) Then the HU should notify an error to user.
+The SDL App can be launched by Voice Recognition with Wakeup Word without pressing down the PTT button. After detecting the Wakeup Word, the user can continuously use the same microphone for Voice Recognition to launch the SDL App. The Wakeup Word is specified in OEM's specification.
+
+
+### 3.3. pronunciationdata of App name (ttsName)
+The ttsName is a pronunciationdata of SDL App name and it is used for specifying the SDL App. The ttsName has multiple pronunciationdata as candidates to identify SDL App name, and the format is in text. The maximum number of pronounciations data that can be registered for one SDL App is up to 100. And, the maximum length of pronunciationdata is 500 characters. SDL App name, pronunciation data, and other App information are obtained from SDL App when the SDL App is registed by `RegisterAppInterface`, then the Voice Recognition command is registered.
```

Comment:
>Lots of rewording to be done here. pronunciationdata is not one word and ttsName should probably just be used in its place, rather than using them as synonyms.

Change the 3.3 title to "ttsName".
Also correct the following:
"The ttsName is a pronounciation data of SDL App name and it is used for specifying the SDL App."-> "The ttsName is used for specifying the SDL App."
Does the pointed out point match?

>The information that this is an array of potential names to listen for doesn't seem to be made clear in the spec, it should probably be included there.

>The 500 character limit is not enforced by the RPC spec, as a note, so that is Toyota-specific

https://smartdevicelink.com/en/docs/hmi/master/common/structs/#hmiapplication
In the SDLC document, "ttsName" is array: true, type: Common.TTSChunk.
The text of TTS Chunk is max length: 500.

Reply:
@jacobkeeler -san, I'd like to revise the documentation, so can you respons? Thank you.